cmake/SearchForStuff.cmake

```diff
@@ -0,0 +1,152 @@
+include (FindPkgConfig)
```

Comment:
we removed this `SearchForStuff.cmake` file when migrating to `ign-cmake2`. We don't want to re-add it. Let's try to figure out which pull request induced a change here and figure out how to apply that change elsewhere (probably the root CMakeLists.txt)

Reply:
looks like the changes were made in https://github.com/gazebosim/sdformat/pull/1042

Reply:
9c81632bca096b86655a0501ca9e47be278ac609